test/test_other.py

```diff
@@ -7921,8 +7923,8 @@ def test_wasm_exceptions_stack_trace(self):
     emcc_args = ['-g', '-fwasm-exceptions']
     self.v8_args.append('--experimental-wasm-eh')
 
-    # Stack trace example for this example code:
-    # exiting due to exception: [object WebAssembly.Exception],Error
+    # Stack trace and message example for this example code:
+    # exiting due to exception: [object WebAssembly.Exception],Error: std::runtime_error, my message
```

Comment:
Unrelated, but I wonder if we could drop the `[object WebAssembly.Exception],` here. It seems to be just unnecessary visual noise.

Reply:
I don't think It's easy to consistently take out that part, because different engines can have different error formats. For example, d8 prints:
```console
main
foo
bar
exiting due to exception: [object WebAssembly.Exception],Error: std::runtime_error,my message
    at __cxa_throw (wasm://wasm/00462286:wasm-function[44]:0x3b0f)
    at bar() (wasm://wasm/00462286:wasm-function[9]:0x98c)
    at foo() (wasm://wasm/00462286:wasm-function[10]:0x992)
    at __original_main (wasm://wasm/00462286:wasm-function[11]:0x9c4)
    at main (wasm://wasm/00462286:wasm-function[12]:0x9e1)
    at test.js:934:22
    at callMain (test.js:1874:15)
    at doRun (test.js:1928:23)
    at run (test.js:1943:5)
```

Whereas Node prints:
```console
/usr/local/google/home/aheejin/test/stack/test.js:147
      throw ex;
      ^

WebAssembly.Exception: std::runtime_error,my message
    at __cxa_throw (wasm://wasm/00462286:wasm-function[44]:0x3b0f)
    at bar() (wasm://wasm/00462286:wasm-function[9]:0x98c)
    at foo() (wasm://wasm/00462286:wasm-function[10]:0x992)
    at __original_main (wasm://wasm/00462286:wasm-function[11]:0x9c4)
    at main (wasm://wasm/00462286:wasm-function[12]:0x9e1)
    at /usr/local/google/home/aheejin/test/stack/test.js:934:22
    at callMain (/usr/local/google/home/aheejin/test/stack/test.js:1874:15)
    at doRun (/usr/local/google/home/aheejin/test/stack/test.js:1928:23)
    at run (/usr/local/google/home/aheejin/test/stack/test.js:1943:5)

Node.js v17.5.0
```

So while both of them have `WebAssembly.Exception`, their printing formats differ, making taking it out in a consistent way difficult. We can just simply replace `WebAssembly.Exception` with an empty string, but not sure if that's better..?

Reply:
Leaving it as is sounds fine for to me then.   We can attempt to refine in a followup perhaps.